app-portage/eix/eix-0.32.1.ebuild

```diff
@@ -26,31 +30,49 @@ DEPEND="${BOTHDEPEND}
 	nls? ( sys-devel/gettext )"
 
 pkg_setup() {
-	case " ${REPLACING_VERSIONS}" in
-	*\ 0.[0-9].*|*\ 0.1[0-9].*|*\ 0.2[0-4].*|*\ 0.25.0*)
-		local eixcache="${EROOT}/var/cache/${PN}"
-		test -f "${eixcache}" && rm -f -- "${eixcache}";;
-	esac
+	# remove stale cache file to prevent collisions
+	local old_cache=${EROOT%/}/var/cache/${PN}
+	if [[ -f ${old_cache} ]]; then
+		rm "${old_cache}" || die
+	fi
```

Comment:
why not just `rm -f "${EROOT%/}/var/cache/${PN}" || die`? Seems much simpler to me